src/css/dark.css

```diff
@@ -110,6 +109,7 @@ input.number:focus {
 		}
 
 		#menubar .menu .options {
+      cursor: pointer;
```

Comment:
should do your best to break it up. it's easy to pile in changes, but it's harder to review